openshift-ci/run_e2e.sh

```diff
@@ -23,64 +25,15 @@ sudo firewall-cmd --zone=libvirt --add-port=3785/udp
 sudo firewall-cmd --zone=libvirt --permanent --add-port=4784/udp
 sudo firewall-cmd --zone=libvirt --add-port=4784/udp
 
-if [[ "$BGP_TYPE" == "frr-k8s" || "$BGP_TYPE" == "frr-k8s-cno" ]]; then
-	MODE_TO_SKIP="FRR-MODE"
-	BGP_MODE="frr-k8s"
-else
-	MODE_TO_SKIP="FRRK8S-MODE"
-	BGP_MODE="frr"
-fi
-
-FRRK8S_NAMESPACE=""
-if [[ "$BGP_TYPE" == "frr-k8s-cno" ]]; then
-  FRRK8S_NAMESPACE="--frr-k8s-namespace=openshift-frr-k8s"
-fi
+go install github.com/onsi/ginkgo/v2/ginkgo@v2.4.0
```

Comment:
that ginkgo version is hardcoded, can we use the go.mod value of the ginkgo version we need?

Reply:
Good catch, this is a leftover, now make ginkgo takes care of it.

Reply:
or not.. it actually does not with metallb tests.